src/main/lib/fs.ts

```
@@ -16,6 +16,12 @@ export interface ReaddirOptions {
     mode?: "flat" | "tree";
 }
 
+// oxlint-disable-next-line no-control-regex
+const WINDOWS_INVALID_CHARS_REGEX = /[<>:"/\\|?*\u0000-\u001F]/g;
```

Biome (2.4.4)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)